hbase-common/src/main/java/org/apache/hadoop/hbase/util/CommonFSUtils.java

```diff
@@ -560,8 +558,7 @@ private static void invokeSetStoragePolicy(final FileSystem fs, final Path path,
     Method m = null;
     Exception toThrow = null;
     try {
-      m = fs.getClass().getDeclaredMethod("setStoragePolicy",
-        new Class<?>[] { Path.class, String.class });
+      m = fs.getClass().getDeclaredMethod("setStoragePolicy", Path.class, String.class);
```

Comment:
branch-1's minimum hadoop version is 2.8.5, so for that branch we can remove this reflection as well. Note to potential further backporters: the method is not in Hadoop 2.7, we'll need this reflection in branch-1.4 and branch-1.3.

Reply:
Removed the reflection part. Will create additional backports for branch-1.4 and branch-1.3.